ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-rollback-stable.yaml

```diff
@@ -0,0 +1,98 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  cli-ocm:
+    name: cli-ocm
+    namespace: ci
+    tag: latest
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
+  rosa-aws-cli:
+    name: rosa-aws-cli
+    namespace: ci
+    tag: latest
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  oldest-supported-z-rollback:
+    release:
+      channel: candidate
+      version: 4.12.0
+  target:
```

Comment:
Sounds we do not need this target release, right?

Reply:
It's referenced in the `toimage` step. So I feel we should either define the `target` release or override `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE` in the job level. As we dropped `OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE`, we need keep `target`, right?

```
dependencies:
  - name: "release:target"
    env: "OPENSHIFT_UPGRADE_RELEASE_IMAGE_OVERRIDE"
```

Reply:
maybe, but we can have a try, if the `target` is needed, then we can keep it.

Or we can define them like:
```
releases:
  latest:
    release:
      channel: candidate
      version: 4.12.0
  oldest-supported-z-rollback:
    release:
      channel: candidate
      version: 4.12.0
  target:
    release:
      architecture: amd64
      channel: candidate
      version: "4.12"
...
  steps:
    cluster_profile: aws-qe
    env:
      BASE_DOMAIN: qe.devcluster.openshift.com
    test:
    - as: set-upgrade-releases
      commands: echo "${RELEASE_IMAGE_TARGET},${RELEASE_IMAGE_Z_ROLLBACK}" > ${SHARED_DIR}/upgrade-edge
      dependencies:
      - env: RELEASE_IMAGE_TARGET
        name: release:target
      - env: RELEASE_IMAGE_Z_ROLLBACK
        name: release:oldest-supported-z-rollback
```
Is this easier to read?